### PR TITLE
feat: Add SendRequest in route service

### LINKED ISSUE #998: Add SendRequest in route service
- Create Route Request

### PR DESCRIPTION
**Pull request by fqutishat**
closes #998

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/didcomm/protocol/route/service.go [hunk 1]
```diff
@@ -66,17 +76,19 @@ type provider interface {
...
 	service.Message
-	routeStore storage.Store
-	outbound   dispatcher.Outbound
-	endpoint   string
-	kms        kms.KeyManager
+	routeStore   storage.Store
+	outbound     dispatcher.Outbound
+	endpoint     string
+	kms          kms.KeyManager
+	vdriRegistry vdriapi.Registry
```
Comment (troyronda) [inline pkg/didcomm/protocol/route/service.go]: or simply `vdr` :)

Comment (fqutishat) [reply pkg/didcomm/protocol/route/service.go]: done

File: pkg/didcomm/protocol/route/service.go [hunk 2]
```diff
@@ -66,6 +76,7 @@ type provider interface {
 	StorageProvider() storage.Provider
 	InboundTransportEndpoint() string
 	KMS() kms.KeyManager
+	VDRIRegistry() vdriapi.Registry
```
Comment (troyronda) [inline pkg/didcomm/protocol/route/service.go]: I assume `VDR()` to match `vdr`

Comment (fqutishat) [reply pkg/didcomm/protocol/route/service.go]: I need to change this method in context

**Pull request merged**